src/main/java/Duke.java

```diff
@@ -1,10 +1,28 @@
 import duke.*;
```

Comment:
It would be a good idea to avoid using wildcard imports, as it is part of the java coding standard we are using for the module.